service-registry/src/test/java/org/apache/servicecomb/serviceregistry/task/TestMicroserviceRegisterTask.java

```diff
@@ -498,12 +508,36 @@ public void testLocalSchemaAndServiceCenterSchemaDiff(@Mocked ServiceRegistryCli
         result = otherMicroservice;
         srClient.getSchemas(anyString);
         result = onlineSchemasHolder;
+        srClient.getSchema(anyString, anyString);
+        result = "abcdefghijklmn";
```

Comment:
multiple lines schema

Reply:
ok..added sample schema file for service center and local